src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay.tsx

```diff
@@ -244,17 +245,19 @@ const StatusDisplay = ({
   const onFinallyRequest = () => {
     refresh?.();
     reloadEntity?.();
-    closeModal();
+    closeModal(ModalId.STATUS_CHANGE);
+    closeModal(ModalId.CHANGE_REQUEST);
```

Comment:
why it has to call twice?

Reply:
The reason closeModal is called twice is because these modals are being opened and managed in different files, each with a unique ID. Specifically:

The modal with ID STATUS_CHANGE is opened and closed in certain files.
The modal with ID CHANGE_REQUEST is managed in a different set of files.
By ensuring that both modals are closed here, we prevent potential issues where one of the modals might remain open if only one ID was specified. This approach ensures that no matter which modal was opened, both are properly closed. This is a safeguard given that we have around five files calling STATUS_CHANGE and another three calling CHANGE_REQUEST.

Additionally, this does not generate any errors or unexpected behavior as the closeModal function gracefully handles the closure of modals based on their IDs.